include/wx/cmndata.h

```diff
@@ -170,20 +171,25 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     void SetMinPage(int v) { m_printMinPage = v; }
     void SetMaxPage(int v) { m_printMaxPage = v; }
     void SetNoCopies(int v) { m_printNoCopies = v; }
+    void SetCurrentPage(int v) { m_printCurrentPage = v; }
     void SetAllPages(bool flag) { m_printAllPages = flag; }
     void SetSelection(bool flag) { m_printSelection = flag; }
     void SetCollate(bool flag) { m_printCollate = flag; }
     void SetPrintToFile(bool flag) { m_printToFile = flag; }
+    void SetSelectCurrentPage(bool flag) { m_printSelectCurrentPage = flag; }
 
     void EnablePrintToFile(bool flag) { m_printEnablePrintToFile = flag; }
     void EnableSelection(bool flag) { m_printEnableSelection = flag; }
     void EnablePageNumbers(bool flag) { m_printEnablePageNumbers = flag; }
     void EnableHelp(bool flag) { m_printEnableHelp = flag; }
+    void EnableCurrentPage(bool flag) { m_printEnableCurrentPage = flag; }
```

Comment:
This is very minor, but I'd put this just below `EnableSelection()` because these functions are related (while it's completely unrelated to `EnableHelp()`).

I'd also consider renaming it to `EnableSelectCurrentPage()` to be consistent with `SetSelectCurrentPage()`. Alternatively, the other function could be renamed to just `SetCurrentPage()` but this would be less clear.

Reply:
Ok, changed.